.travis.yml

```diff
@@ -167,24 +167,23 @@ script:
     - install_run $PLAT
 
 after_success:
-    # trigger an upload to the shared ecosystem
-    # infrastructure at: https://anaconda.org/scipy-wheels-nightly
-    # for cron jobs only (restricted to master branch once
-    # per week)
-    # SCIPY_WHEELS_NIGHTLY is a secret token
-    # used in Travis CI config, originally
-    # generated at anaconda.org for scipy-wheels-nightly
-    - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+    # upload wheels for pushes and merges.
+    - if [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
Do we really want to have new wheels in the nightly repo for a reason other than the weekly crons? I thought the weekly cadence was intentional--since the wheels repo can go quiet for > 1 month at a time, if we really wanted to consistently provide i.e., daily updates to the bleeding edge "nightly" wheels wouldn't we want that to be reflected in the frequency of the crons?